core/src/main/java/org/apache/iceberg/BaseMetastoreTableOperations.java

```diff
@@ -414,16 +414,20 @@ private void deleteRemovedMetadataFiles(TableMetadata base, TableMetadata metada
             TableProperties.METADATA_DELETE_AFTER_COMMIT_ENABLED_DEFAULT);
 
     if (deleteAfterCommit) {
-      Set<TableMetadata.MetadataLogEntry> removedPreviousMetadataFiles =
-          Sets.newHashSet(base.previousFiles());
-      removedPreviousMetadataFiles.removeAll(metadata.previousFiles());
-      Tasks.foreach(removedPreviousMetadataFiles)
+      Set<TableMetadata.MetadataLogEntry> filesToDelete = Sets.newHashSet(base.previousFiles());
```

Comment:
It doesn't look like there are behavior changes in this commit, just variable renames? While I like better variable names, we don't want to introduce code churn just to rename things.

Reply:
yes this was only a variable rename to make things a bit clearer to understand when reading this part of the code as `removedPreviousMetadataFiles` was a bit confusing